recipes/taxor/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: taxor
+  version: {{ version }}
+
+source:
+  url: https://github.com/JensUweUlrich/Taxor/releases/download/{{ version }}/taxor.tar.gz # [linux]
+  md5: 8e5476aadc3fecab580e8b1b73ba3dff # [linux]
```

Comment:
The usage of `# [linux]` makes me think that you need to add `build.skip: True # [osx]` below

Reply:
Thanks for pointing that out. I included the recommended skipping of building for osx